.vscode/launch.json

```diff
@@ -29,6 +29,39 @@
 			"env": {
 				"ASPIRE_CLI_STOP_ON_ENTRY": "true"
 			},
+      "cwd": "${workspaceFolder}/extension"
```

Comment:
I added a bunch of configurations to make debugging the apphost & cli easier, which was used in testing this branch